src/main/java/com/google/codeu/data/UserMarker.java

```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
+public class UserMarker {
+
+    private double lat;
```

Comment:
Can you please explain what lat, lng and content variables are please?

Reply:
Lat -> Latitude
Lng -> Longitude
content -> users message on the marker